Spokestack/NLUTensorflowTokenizer.swift

```diff
@@ -82,31 +82,33 @@ internal struct WordpieceTokenizer: Tokenizer {
         }
     }
     
-    /// Decode and detokenize the encoded tokens into a space-separated string.
-    /// - Parameter tokens: The encoded tokens to decode and detokenize.
-    func detokenize(_ tokens: [String]) throws -> String {
-        return tokens.reduce("", { (result, s) in
+    func decode(_ tokens: [String]) ->  [String] {
```

Comment:
This doesn't really decode (turn IDs into tokens); it's more the Wordpiece-specific half of detokenization.